test/sengoku/ai/random_test.exs

```diff
@@ -13,7 +13,8 @@ defmodule Sengoku.AI.RandomTest do
       tiles: %{
         1 => %Tile{owner: 1, units: 1, neighbors: [2]},
         2 => %Tile{owner: nil, units: 1, neighbors: [1]}
-      }
+      },
+      required_move: nil
```

Comment:
I probably need some kind of text fixture data to reduce this duplication, but I worry about introducing [mystery guests](https://robots.thoughtbot.com/mystery-guest)...